cfme/v2v/migrations.py

```diff
@@ -249,6 +256,58 @@ class AddMigrationPlanView(View):
     next_btn = Button('Next')
     cancel_btn = Button('Cancel')
 
+    @View.nested
+    class general(View):  # noqa
+        infra_map = BootstrapSelect('infrastructure_mapping')
+        name = TextInput(name='name')
+        description = TextInput(name='description')
+        select_vm = RadioGroup('.//*[contains(@id,"vm_choice_radio")]')
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button('Import')
+        importcsv = Button('Import CSV')
+        hidden_field = HiddenFileInput(locator='.//*[contains(@accept,".csv")]')
+        table = Table('.//*[contains(@class, "container-fluid")]/table',
+                      column_widgets={"Select": Checkbox(locator=".//input")})
+        filter_by_dropdown = SelectorDropdown('id', 'filterFieldTypeMenu')
+        search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+        clear_filters = Text(".//a[text()='Clear All Filters']")
```

Comment:
Tried `Search` View but I did not liked it since only two widgets matching after adjustment (search_input and clear_button) and functions also not matching.

Written code working properly with following widget - 
![screencast from 2018-06-19 13-36-10](https://user-images.githubusercontent.com/10824880/41584786-d2732332-73c5-11e8-8b0b-1d644d3cd08e.gif)